operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -330,9 +332,11 @@ private Step patchCurrentPod(V1Pod currentPod, Step next) {
     KubernetesUtils.addPatches(
         patchBuilder, "/metadata/annotations/", getAnnotations(currentPod), getPodAnnotations());
 
-    return new CallBuilder()
-        .patchPodAsync(getPodName(), getNamespace(), getDomainUid(),
-            new V1Patch(patchBuilder.build().toString()), patchResponse(next));
+    return Step.chain(
```

Comment:
would the domain status be updated when we stop a server, or scale down a cluster?

Reply:
Once the domain is started, the operator periodically checks the server's state and health condition, and update the domain status/condition accordingly. Right now, we don't have a progressing REASON for cluster scaling down; the condition we are adding here is for managed server starting only. 
I am looking into the scaling down code path to see if we need to add additional Progressing step or not.

Reply:
I added ProgressingStep to scaling down code path to match what we had without the changes in this PR. I thought about adding a new reason (for example "managedServersScalingDown") for this, but I decided to use what we used before this change. Ryan, your thoughts on this?